manuscript/Draft-notes/External-images.md

```diff
@@ -9,9 +9,8 @@ type: draft
 
 http://blog.diniscruz.com/2016/11/presentation-hacking-portugal-and.html
 
-Slide 1:
 
-Wikipedia (2016) Caravela in sail [image online], available: https://upload.wikimedia.org/wikipedia/commons/a/a2/Caravela_Vera_Cruz_no_rio_Tejo.jpg [accessed 14 Nov 2016].
+Cover image: Wikipedia (2016) Caravela in sail [image online], available: https://en.wikipedia.org/wiki/Portuguese_inventions#/media/File:Caravela_Vera_Cruz_no_rio_Tejo.jpg [accessed 14 Nov 2016].
 
```

Comment:
Hi @Ambg05 , can you move this reference to the Cover Image to a file in the manuscript folder? (so that it is picked up by the book generation engine)